lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -551,10 +551,8 @@ def process(self, forecast_probabilities, no_of_percentiles=None,
                 "GeneratePercentilesFromProbabilities")
 
         threshold_coord = find_threshold_coordinate(forecast_probabilities)
-        forecast_probabilities = concatenate_cubes(
-            forecast_probabilities,
-            coords_to_slice_over=threshold_coord,
-            coordinates_for_association=[])
+        if not isinstance(forecast_probabilities, iris.cube.Cube):
+            forecast_probabilities = forecast_probabilities.concatenate_cubes()
```

Comment:
Is this to address a current use case or is it just a hedge against a cubelist making it here somehow?

Reply:
The reason for this particular change is that the IMPROVER concatenate_cubes functionality also strips var names from cubes, as this previously caused some problems in that it was my previous observation that var_names were added upon saving, but were not present prior to saving and hence, to lead to some mismatches in coordinates depending upon they had just been loaded from a file or had been constructed in memory. In this case, stripping the var_names was causing a problem in ApplyCoefficientsFromEnsembleCalibration, as coordinate that should be the same were differing only in their var_names. However as for the bigger picture question, we quite probably do not actually need to support cubelists in this plugin now, so we could do a little bit of a clean-up in a follow-on ticket.